py/embedded-server/deephaven_server/server.py

```diff
@@ -120,22 +120,33 @@ def __init__(
         port: Optional[int] = None,
         jvm_args: Optional[List[str]] = None,
         extra_classpath: Optional[List[str]] = None,
+        default_jvm_args: Optional[List[str]] = None,
     ):
         """
         Creates a Deephaven embedded server. Only one instance can be created at this time.
+
+        Args:
+            host (Optional[str]): The host to bind the server to, defaults to None, meaning local host.
+            port (Optional[int]): The port to bind the server to, defaults to None, meaning 10000.
+            jvm_args (Optional[List[str]]): The common, user specific JVM arguments, such as JVM heap size, the
+                authentication handler to use, and other related JVM options. Defaults to None.
+            extra_classpath (Optional[List[str]]): The extra classpath to use.
+            default_jvm_args (Optional[List[str]]): The advanced JVM arguments to use instead of the default ones that
+                Deephaven recommends, such as a specific garbage collector and related tuning parameters, or whether to
+                let Python or Java to handle signals. Defaults to None.
```

Comment:
I thought so before, when the user had to access that to change things, but since they can override them _or_ replace them (or both, though not quite sure why you'd do both), I'd tend to say we document them, but not expose them? Is there an equivalent of the javadoc tag `@value` in pydocs?

Reply:
I'm not aware of one, but @jmao-denver may know of one.

I asked the question about making the defaults accessible only because they can be overridden.  They may want to see what the baseline values are before constructing an override.